tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1334,9 +1316,7 @@ public function testListDocumentsLimitAndOffset(array $data): array
             'content-type' => 'application/json',
             'x-appwrite-project' => $this->getProject()['$id'],
         ], $this->getHeaders()), [
-            'limit' => 1,
-            'orderAttributes' => ['releaseYear'],
-            'orderTypes' => ['ASC'],
+            'queries' => [ 'limit(1)', 'orderAsc("releaseYear")' ],
```

Comment:
@gewenyu99 We should make it clear in docs that ordering matters for `orderDesc` and `orderAsc` queries, but not any others (correct me if I'm wrong @Meldiron @stnguyen90). E.g:

```php
[ 'orderAsc("title")', 'limit(1)', 'orderAsc("releaseYear")' ]
```
and
```php
['limit(1)', 'orderAsc("title")', 'orderAsc("releaseYear")' ]
```

Would produce the same results, however:

```php
['limit(1)', 'orderAsc("releaseYear")', 'orderAsc("title")' ]
```

Would be different, as ordering is cumulative.